RecoBTag/FeatureTools/src/deep_helpers.cc

```diff
@@ -1,7 +1,20 @@
 #include "RecoBTag/FeatureTools/interface/deep_helpers.h"
 #include "DataFormats/PatCandidates/interface/PackedCandidate.h"
 #include "FWCore/Utilities/interface/isFinite.h"
-
+#include "TrackingTools/Records/interface/TransientTrackRecord.h"
+#include "TrackingTools/TrajectoryState/interface/TrajectoryStateOnSurface.h"
+#include "TrackingTools/GeomPropagators/interface/AnalyticalTrajectoryExtrapolatorToLine.h"
+#include "TrackingTools/GeomPropagators/interface/AnalyticalImpactPointExtrapolator.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/Candidate/interface/VertexCompositePtrCandidate.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "TLorentzVector.h"
+#include "DataFormats/Math/interface/deltaR.h"
```

Comment:
Is this include needed? If not, please remove (and also check the other ones)

Reply:
Yes this is necessary for obtaining the minimum deltaR of a track w.r.t. the SVs, the minimum distance of a track w.r.t. the SVs and the subjet deltaR